quickstart-python/README.md

```diff
@@ -40,11 +40,17 @@ The application will start running at `http://0.0.0.0:8080`.
 
 This app showcases how to use various Monday app functionalities, including:
 
-* Using Monday apps storage
-* Utilizing Monday apps Secrets / Environment variables
+### Monday Code App Features
```

Comment:
Monday code app features? do you mean **app features**? or **monday code features**?

Reply:
correct
monday code features